plugin.yaml

```diff
@@ -2405,6 +2405,9 @@ node_types:
       <<: *client_config
       <<: *resource_id
       <<: *tags_property
+      cloudify_tag:
+        description: an automatic tag to identify the ec2 instance
+        type: boolean
```

Comment:
In the code, you put `cloudify_tagging`. Which is it, `cloudify_tagging` or `cloudify_tags`? We should also add this to `cloudify.nodes.aws.ec2.BaseType` and other ones as well.